src/test/java/bio/terra/service/snapshotbuilder/utils/CriteriaQueryBuilderTest.java

```diff
@@ -251,7 +269,7 @@ private static SnapshotBuilderProgramDataListCriteria generateListCriteria() {
     return (SnapshotBuilderProgramDataListCriteria)
         new SnapshotBuilderProgramDataListCriteria()
             .values(List.of(new BigDecimal(0), new BigDecimal(1), new BigDecimal(2)))
-            .id(new BigDecimal(0))
+            .id(1)
```

Comment:
Do we want to change the values from number to integer? And does this change from 0 to 1 matter?

Reply:
Ah, good catch - yes, the values should also be ints. I will change. The change from 0->1 does matter, since we now look things up with the ID, so the list and range IDs need to be distinct.